### PR TITLE
Ziti dump with path

### PR DESCRIPTION
**Pull request by mary-dcouto**
Fix for the ziti dump issue. Ziti dump files will be generated when the user clicks on the feedback option. ZET requires an extra parameter for the ziti dump command, i.e. the DumpPath where the ziti dumps will be placed.

### CODE REVIEW

File: ziti-edge-tunnel/build.bat [hunk 1]
```diff
@@ -14,7 +14,7 @@ REM See the License for the specific language governing permissions and
 REM limitations under the License.
 REM
 SET REPO_URL=https://github.com/openziti/ziti-tunnel-sdk-c.git
-SET ZITI_TUNNEL_REPO_BRANCH=memory_check
+SET ZITI_TUNNEL_REPO_BRANCH=desktop-edge-win
```
Comment (dovholuknf) [inline ziti-edge-tunnel/build.bat:17]: intentional change?

**Comment by dovholuknf** [comment]
It seems like it might make sense to rename the target to "release-1.12.x" branch?

**Comment by dovholuknf** [comment]
does the zip file get placed into the same location too?  @JeremyTellier is it easy enough to prompt for the location to put the zip file - and then STOP opening an email automatically? i'd really like to close this issue out at the same time - seems like a double word score here: https://github.com/openziti/desktop-edge-win/issues/396

File: release-notes.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+# Release 1.12.1
+
+## What's New
+* The ziti tunnel service is changed to ziti-edge-tunnel service. This service uses the 0.17.x t-sdk version, so the new capabilities like srv support that are added to the tunnel sdk will be available in ziti-edge-tunnel service.
```
Comment (dovholuknf) [inline release-notes.md:4]: are there any other things that are going to be new to add here? calling out srv support is cool but probably shouldn't be mentioned until it works/is tested (i don't think it's tested yet?)

Comment (mary-dcouto) [reply release-notes.md:4]: I will remove srv from this comment

**Comment by mary-dcouto** [comment]
> does the zip file get placed into the same location too? 

Yes, the zip file will get placed in the same location

**Comment by mary-dcouto** [comment]
> It seems like it might make sense to rename the target to "release-1.12.x" branch?

I renamed the base branch to release-1.12.x

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**